frontend/src/service/musiComApiService.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios'
+
+const baseUrl = '/api'
```

Comment:
you may want to use a more descriptive API path here, because it does not indicate what kind of information you can get from this endpoint

Reply:
I will have a look after I have some more endpoints how to structure them...